checker/check_runner.go

```diff
@@ -100,6 +100,12 @@ func (r *Runner) Run(ctx context.Context, c Check) CheckResult {
 	if err != nil {
 		panic(err)
 	}
+
+	ctx, err = tag.New(ctx, tag.Upsert(stats.RepoHost, r.CheckRequest.Repo.Host()))
+	if err != nil {
+		panic(err)
```

Comment:
Lets avoid `panic`ing. Can you change the one above too? You can find a non-`panic` approach in `logStats`